#Parameters related to visualization of training resp. recognition results.

#Where to show our visualization results.
visualization_topic: /ism_results_visualization
